js/main.js

```diff
@@ -0,0 +1,325 @@
+  linkToLive: 'https://github.com/solorics',
+  btn: 'See Source',
+  side: ['Next Project', 'Previous Project'],
+  icons: ['./assets/icons/Vector.png', './assets/icons/Union.png'],
+  btnForPopup: ['See Live', 'See Source'],
+  nextPrevious: ['./assets/icons/pre.png', './assets/icons/next.png'],
+}, {
+  id: '6',
+  class: 'item10',
+  title: 'Project name goes here',
+  features: ['HTML/CSS', 'Ruby on Rails', 'JavaScript'],
+  image: './assets/images/Rectangle26.svg',
+  img: './assets/images/Union.svg',
+  sideArrows: ['./assets/icons/left.png', './assets/icons/right.png'],
+  sideImage: [
+    './assets/images/Rectangle26.svg',
+    './assets/images/Rectangle26.svg',
+    './assets/images/Rectangle26.svg',
+    './assets/images/Rectangle26.svg',
+  ],
+  description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+enim ad minim veniam, quis nostrud exercitation ullamco laboris
+<br /><br />
+nisi Ut aliquip ex ea commodo consequat. Lorem ipsum dolor sit
+amet, consectetur adipiscing elit, sed do eiusmod tempor
+incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+veniam, quis nostrud exercitation ullamco laboris nisi.`,
+  linkToSource: 'https://solorics.github.io/Portfolio-Mobile-FIrst/',
+  linkToLive: 'https://github.com/solorics',
+  btn: 'See Source',
+  side: ['Next Project', 'Previous Project'],
+  icons: ['./assets/icons/Vector.png', './assets/icons/Union.png'],
+  btnForPopup: ['See Live', 'See Source'],
+  nextPrevious: ['./assets/icons/pre.png', './assets/icons/next.png'],
+},
+];
+
+const projectList = document.querySelector('#portfolio ul.grid');
+
+function loadProjects() {
+  for (let i = 0; i < projects.length; i += 1) {
+    const projectItemtemplate = `
+    <li class="project-item ${projects[i].class}">
+      <img
+        class="project-img"
+        src="${projects[i].image}"
+        alt="laptop"
+      />
+      <div class="container container-information hover">
+        <h2 class="heading">${projects[i].title}</h2>
+        <ul class="features">
+          <li class="feature">${projects[i].features[0]}</li>
+          <li class="feature">${projects[i].features[1]}</li>
+          <li class="feature">${projects[i].features[2]}</li>
+        </ul>
+        <button data-id="${projects[i].id}" class="c-projects">
+          See this project &rarr;
+        </button>
+      </div>
+    </li>
+  `;
+
+    projectList.innerHTML += projectItemtemplate;
+  }
+}
+
+window.addEventListener('load', loadProjects());
+
+function createModal(id) {
+  const pCardTemplate = `
+  <div class="overlay hidden">
+      <article class="modal container">
+        <div class="modal__header">
+          <div class="modal__header_title flex space-between">
+            <h1 class="top_title">${projects[id].title}</h1>
+            <button type="button" class="top_btn">
+              &#x2716
+            </button>
```

Comment:
- [ ] [OPTIONAL] Kindly, your project should be responsive, check your modal window between 768px and 1100px, 🎉 there appears a space that shows the main page, it should be removed to have a better user experience, and sticks with the original design 🦖.

![image](https://user-images.githubusercontent.com/91301423/203186841-0d4b527f-be8c-4c5f-b583-bb19d2f10f2c.png)